.circleci/main.yml

```diff
@@ -434,6 +434,20 @@ jobs:
       - checkout
       - run: sudo npm install -g prettier
       - run: prettier -c etc/helm/pachyderm/values.yaml .circleci/config.yml .circleci/main.yml
+  jsonnet-lint:
+    resource_class: small
+    machine:
+      image: << pipeline.parameters.machine_image >>
+    steps:
+      - checkout
+      - run:
+          name: Install go-jsonnet
+          command: |
+            go install github.com/google/go-jsonnet/cmd/jsonnet-lint@latest
+      - run:
+          name: lint all jsonnet files
+          command: |
+            find . -maxdepth 10 -name \*.jsonnet -exec jsonnet-lint {} \;
```

Comment:
I think that find exits with success even if the exec command fails:

```
$ find . -exec /bin/false ';' ; echo $?
0
```
We might have to find some way to make this exit non-zero, otherwise we won't actually be testing anything.

Reply:
LOL github ate one variant of this comment, now it's back.

Reply:
yup. you were right. good catch. tried it with a pipefail instead.[ here is example run where I broke something on purpose and it failed as expected](https://app.circleci.com/pipelines/github/pachyderm/pachyderm/13402/workflows/9741d9ef-9ee4-4a8b-81dd-4fd552b363d2/jobs/190136)